contrib/opencensus-ext-azure/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+## 1.0.0
+Released 2019-09-30
+
+- Standard Metrics - Incoming Requests Execution Time
```

Comment:
"Incoming Requests Execution Time" is a term of art and should get capitalized like this?